src/transforms/spark-jobs/geojson_to_db/main.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
@safiyahs This script needs to use native spark APIs to insert the records to Postgres. The current approach will use only the driver node of the Spark cluster which would be a overkill. 

@mandarinamdar  Thoughts?